apollo-client/src/main/java/io/logz/apollo/clients/GenericApolloClient.java

```diff
@@ -188,8 +188,9 @@ String getUserName() {
 
     private String getFullUrlWithToken(String url) {
         String urlPrefix = prefix.isPresent() ? "/" + prefix.get() : "";
-        String tokenPostfix = StringUtils.isNotBlank(token) ? "?_token=" + token : "";
-        return protocol + "://" + hostname + ":" + port + urlPrefix + url + tokenPostfix;
+        String tokenPrefix = url.contains("?") ? "&" : "?";
```

Comment:
Why? What is the use?

Reply:
if someone wants to send query params
for example `/api?x=1&y=2`
without this fix resulting request will look like the following: `/api?x=1&y=2?_token=****`
making it invalid. question mark appears twice in this request.